examples/kerberos.rs

```diff
@@ -53,89 +39,116 @@ fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     }
 }
 
-fn get_cred_handle(kerberos: &mut Kerberos) -> AcquireCredentialsHandleResult<Option<CredentialsBuffers>> {
-    let username = "user@your.domain".to_string();
-    let password = "user's_password".to_string();
-    let identity = sspi::AuthIdentity {
-        username,
-        password: password.into(),
-        domain: None,
+#[cfg(feature = "network_client")]
```

Comment:
You can put this at the top of the file:
```
#![cfg(feature = "network_client")]
```

Reply:
the problem is that ```cargo test``` needs to see the main function present in the file. if I put it on top, it will not pass ```cargo test```

Reply:
You’re right! Actually, in this case you should remove the`#[cfg]`s altogether, and add this at the bottom of the `Cargo.toml` instead:
```rust
[[example]]
name = "kerberos"
required-features = ["network_client"]
```

This specifies that this example needs the "network_client" feature enabled in order to be compiled and it will be ignored if the feature is not available.